food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -10,196 +10,368 @@
+              </else-if>
+              <else-if type="thesis">
+                <!-- thesis -->
+                <text variable="title" font-style="italic"/>
+                <text macro="publisher-and-place"/>
+                <text variable="genre" prefix=" (" suffix=")"/>
+                <text macro="Available-at"/>
+              </else-if>
+              <else-if type="song motion_picture article" match="any">
+                <!-- eBook, audio recording, film, and video recording in Zotero; film and computer program in Mendeley -->
+                <choose>
+                  <if match="any" variable="medium genre">
+                    <group>
+                      <text variable="title" font-style="italic"/>
+                      <text variable="medium" prefix=" [" suffix="]"/>
+                      <!-- Can put "audio / video / Kindle edition" in "Format" field in Zotero. -->
+                      <text variable="genre" prefix=" [" suffix="]"/>
+                      <!-- Can put "video / CD&#8211;ROM" in "Genre" field in Mendeley. -->
+                    </group>
+                  </if>
+                  <else>
+                    <text variable="title"/>
+                  </else>
+                </choose>
+                <text macro="publisher-and-place" suffix="."/>
+                <text macro="Cite-and-URL"/>
+              </else-if>
+              <else-if type="manuscript">
+                <!-- manuscript -->
+                <text variable="title"/>
+                <text variable="genre" suffix="."/>
+                <text variable="publisher-place" suffix="."/>
+                <text macro="number_of_pages-label"/>
+                <text macro="Available-at"/>
+              </else-if>
+              <else-if type="speech" match="any">
+                <!-- presentation -->
+                <text variable="title"/>
+                <group suffix=".">
+                  <text term="presented at" prefix="Speech "/>
+                  <text variable="event" prefix=" "/>
+                  <date form="text" variable="issued" prefix=", "/>
+                  <text variable="publisher-place" prefix=", "/>
+                </group>
+                <text macro="Cite-and-URL"/>
+              </else-if>
+              <else-if type="paper-conference">
+                <!-- conference paper -->
+                <text variable="title"/>
+                <group>
+                  <text term="in" text-case="capitalize-first" font-style="italic"/>
+                  <text macro="editor" prefix=" "/>
+                </group>
+                <text variable="container-title" font-style="italic"/>
+                <group suffix=".">
+                  <label suffix=" " variable="page" form="short"/>
+                  <text variable="page"/>
+                </group>
+                <text variable="collection-title"/>
+                <group suffix=".">
+                  <text term="presented at" prefix="Paper "/>
```

Comment:
There's a "Speech presented at" before so it's kind of difficult. But your point of localization is good. I'll define the term and persuade the editor to change the output of the presentation type to "Paper presented at" in the style guide document. Presentation is very rarely used, anyway.